addons/website_rating/controllers/website_mail.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from werkzeug.exceptions import NotFound
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.website_mail.controllers.main import WebsiteMail
+
+
+class WebsiteRating(WebsiteMail):
+
+    def _website_message_post(self, res_model, res_id, message, **kw):
```

Comment:
It seems strange this behavior is implemented here. Shouldn't it be implemented at model level, somewhere like in an override of mail.thread in rating ?